task/sensor/src/main.rs

```diff
@@ -98,6 +126,7 @@ fn main() -> ! {
 
     let mut server = ServerImpl {
         data: [Reading::Absent; NUM_SENSORS],
+        nerrors: [0; NUM_SENSORS],
```

Comment:
This is going to conflict with https://github.com/oxidecomputer/hubris/pull/1018 (should be an easy fix)